platform/security/certificate/security-ocsp-checker/src/main/java/org/codice/ddf/security/ocsp/checker/OcspChecker.java

```diff
@@ -446,6 +474,191 @@ private void postErrorEvent(String errorMessage) {
     LOGGER.debug(errorMessage);
   }
 
+  private void logRequest(OCSPReq ocspRequest) {
+    StringBuffer logbuff = new StringBuffer();
```

Comment:
Generally we should be using `StringBuilder` instead of `StringBuffer` in single-threaded situations.

Reply:
Fixed.